test/unit/deferred.js

```diff
@@ -542,28 +542,30 @@ QUnit.test( "jQuery.Deferred.then - spec compatibility", function( assert ) {
 	} catch ( _ ) {}
 } );
 
+// Test fails in IE9 but is skipped there because console is not active
 QUnit[ window.console ? "test" : "skip" ]( "jQuery.Deferred.exceptionHook", function( assert ) {
 
-	assert.expect( 1 );
+	assert.expect( 2 );
 
 	var done = assert.async(),
 		defer = jQuery.Deferred(),
 		oldWarn = window.console.warn;
 
-	window.console.warn = function( msg ) {
+	window.console.warn = function() {
 
 		// Support: Chrome <=41 only
```

Comment:
Yeah, although technically that shouldn't matter as Android 5.0 has both auto-updatable browser (Chrome) & WebView. It's just that BrowserStack doesn't update Chrome in Android images (I'm not sure if that's even possible in emulators). The solution would be to move to Android 6.0 and drop 5.0 as we're really interested in testing Chrome for Android, not specific Android versions (it used to be different as Android Browser versions were tied to Android versions as is still the case on iOS) but last time I checked local testing didn't work on Android 6.0 on BrowserStack. I'll need to re-check, if it worked we'd be able to drop these workarounds.